src/f_wipe.c

```diff
@@ -354,7 +356,10 @@ void F_RunWipe(UINT8 wipetype, boolean drawMenu)
 
 		// wait loop
 		while (!((nowtime = I_GetTime()) - lastwipetic))
-			I_Sleep();
+		{
+			I_Sleep(cv_sleep.value);
```

Comment:
```suggestion
			if (cv_sleep.value > 0)
				I_Sleep(cv_sleep.value);
```

This should fix the freeze on launch.